typings/next-routes.d.ts

```diff
@@ -44,11 +44,14 @@ export interface Registry {
   Router: Router;
 }
 
-export default class Routes implements Registry {
+export class Routes implements Registry {
   getRequestHandler(app: Server, custom?: HTTPHandler): HTTPHandler;
   add(name: string, pattern?: string, page?: string): this;
   add(pattern: string, page: string): this;
   add(options: { name: string; pattern?: string; page?: string }): this;
   Link: ComponentType<LinkProps>;
   Router: Router;
 }
+
+declare const _default: (opt: any) => Routes
+export default _default
```

Comment:
instead of the above two lines prefer the following:
```ts
export interface RoutesOptions {
  Link: ComponentType<LinkProps>;
  Router: Router;
}

export default function makeRoutesRegistry(options?: RoutesOptions): Routes;
```

Reply:
The reasons to prefer include options are explicitly typed instead of `any` and the constructor function is nicely named